app/models/imports/podcast_timings_import.rb

```diff
@@ -93,6 +97,11 @@ def import!
         ei.timings = timings
         ei.save!
         ei.import_later
+
+        breaks = ei.parse_timings
+        unless breaks.nil?
+          ad_breaks[breaks.count] = ad_breaks[breaks.count].to_i + 1
```

Comment:
This is a UI nice to have - on import, we count the number of episodes with 0/1/2/etc breaks.  And display it in the status box.

![image](https://github.com/PRX/feeder.prx.org/assets/1410587/b7078fc3-d6e5-43ae-9789-1f6f1b6c6aad)